The file analyze_user_genomes.sh classifies user genomes according to their
chitinase-linked gene content, and assigns user genomes putative ecological
roles in chitin degrading bacterial communities

This script assumes the user has:
  *   A working R installation
  *   A working python installation
  *   Run-dbcan (specifically the train_many_organisms_many_families.py script)
      *   Run-dbcan can be installed using the command: pip install run-dbcan
  *   The following tools are in the users PATH
      *   Mmseqs
      *   Awk (preferably mawk aliased as awk)
      *   Hmmer
      *   gtdb-tk (can be installed using the command pip install gtdbtk)
        *   prodigal (for gtdb-tk)
        *   pplacer (for gtdb-tk)
        *   FastANI (for gtdb-tk)
        *   FastTree (for gtdb-tk)
        *   Mash (for gtdb-tk)

(*) The first argument is a path to the directory which contains a separate 
      protein fasta file for each genome the user wishes to analyze
(*) The second argument is a path to the directory which contains genomic
      fna files



the folder figure_scripts contains scripts to reproduce analysis and figures from:

Public good exploitation in natural bacterioplankton communities.
Shaul Pollak, Matti Gralka, Yuya Sato, Julia Schwartzman, Lu Lu, Otto X. Cordero.

